<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDialog</class>
 <widget class="QDialog" name="QDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>741</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>HoloLens MDL2 Assets</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Search tool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>window.png</normaloff>window.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>500</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>URL</string>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>検索文字列</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>濾過文字列</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="edit_key">
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>HoloLens MDL2 Assets</family>
               <pointsize>16</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filter_key">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>HoloLens MDL2 Assets</family>
               <pointsize>16</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btn_search">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>90</height>
            </size>
           </property>
           <property name="toolTip">
            <string>検索</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>search.png</normaloff>search.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cancel">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>80</height>
            </size>
           </property>
           <property name="toolTip">
            <string>終了</string>
           </property>
           <property name="statusTip">
            <string>終了</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>quit.png</normaloff>quit.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Georgia</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>複数のサイトを同時に検索するためのツール</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>658</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lbl_time">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>07/26/2021</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>QDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_search</sender>
   <signal>clicked()</signal>
   <receiver>QDialog</receiver>
   <slot>showResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_key</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QDialog</receiver>
   <slot>changeSearchKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeSearchKey()</slot>
  <slot>selectTableItem(int,int)</slot>
 </slots>
</ui>
